src/test/java/org/jenkinsci/plugins/github_branch_source/GitSCMSourceBase.java

```diff
@@ -78,11 +78,8 @@ public void prepareMockGitHub() throws Exception {
         githubRaw.stubFor(get(urlMatching(".*")).atPriority(10)
                 .willReturn(aResponse().proxiedFrom("https://raw.githubusercontent.com/")));
 
-        if (source.isConfiguredByUrl()) {
-            source = new GitHubSCMSource("cloudbeers", "yolo", "http://localhost:" + githubApi.port() + "/cloudbeers/yolo", true);
-        } else {
-            source.setApiUri("http://localhost:" + githubApi.port());
-        }
+        // force apiUri to point to test server
+        source.forceApiUri("http://localhost:" + githubApi.port());
```

Comment:
Related to parent update somehow?

Reply:
I'm not sure how these tests ever passed and they were failing on my machine.   I had to update to get to a passing build.